django_app/settings.py

```diff
@@ -161,6 +161,16 @@
     },
 ]
 
+# DRF global settings
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework.authentication.TokenAuthentication",
+    ],
+    "DEFAULT_PERMISSION_CLASSES": [
+        "rest_framework.permissions.IsAuthenticated",
+        "rest_framework.permissions.IsAdminUser",
```

Comment:
Please use IsAdminUser permission classes to specific API, this default permission class will deny permission to any user.

Reply:
Having IsAdminUser and IsAuthenticated as default permission classes wouldn't work, as all non admin users would be denied permission by default, hence IsAdminUser should be used on specific API classes/methods that needs only admin interaction.